subsys/spm/secure_services.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <cortex_m/tz.h>
+
+#if defined(CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS)
+
+extern void sys_arch_reboot(int type);
```

Comment:
Should we have a Kconfig per secure function?

Reply:
We would eventually have this I think, it can be added later.

Note that, doing this properly (i.e. avoiding having to manually match kconfigs between SPM and app) sort of needs Kconfig inheritance, so I'm not sure what the best way is.